package.json

```diff
@@ -88,6 +88,15 @@
                     "default": "false",
                     "description": "Specifies whether or not to generate cleanup when using react-testing-library"
                 },
+                "createReactComponent.fileExtension": {
+                    "type": "string",
+                    "enum": [
+                        "withX",
+                        "withoutX"
+                    ],
+                    "default": "withX",
+                    "description": "Specifies the extension type of files: .ie 'withX' would create 'MyComponent.tsx' or 'MyComponent.jsx'. 'withoutX' would create 'MyComponent.ts' or 'MyComponent.js' files"
+                },
```

Comment:
Could you please add some documentation for this setting in the README? 😄 
https://github.com/yeet-bix/create-react-component-vscode/#settings

Reply:
Done!